src/components/tasks/card/index.tsx

```diff
@@ -26,9 +29,28 @@ const Card: FC<Props> = ({
   const statusRedList = [BLOCKED];
   const statusNotOverDueList = [COMPLETED, VERIFIED, AVAILABLE];
   const cardDetails = content;
-  const [assigneeProfilePic, setAssigneeProfilePic] = useState(
-    `${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}/${cardDetails.assignee}/img.png`,
-  );
+  const [assigneeProfilePic, setAssigneeProfilePic] = useState(DUMMY_PROFILE_PATH);
+
+  const memoizeAssignProfilePic = useCallback(async () => {
```

Comment:
If it is repeating , could we think about having a custom hook?

Reply:
yeah, It can be done.